components/organisms/colorDropDown/__tests__/stories.js

```diff
@@ -0,0 +1,60 @@
+/**
```

Comment:
@mhsnprvr2 This is not an organism at all - I did say stick it as a file inside dropdown folder as its connected :) - re-work that please - the stories can then be connected also....

Reply:
ok